config/inference_router_config.json

```
@@ -0,0 +1,45 @@
+{
+  "basic": {},
+  "health_check": {
+    "check_frequency": "10s",
+    "consecutive_success": 2,
+    "consecutive_failure": 2
+  },
+  "servers": [
+    {
+      "name": "main-server",
+      "addr": "[::]:8443",
+      "locations": [
+        "inference",
+        "up"
+      ],
+      "threads": 0,
+      "prometheus_metrics": "/metrics"
```

This config setting is named `metrics_path`

```suggestion
      "metrics_path": "/metrics"
```

---

Done.